src/components/TokenCard/TokenCard.tsx

```diff
@@ -101,11 +102,24 @@ export default function TokenCard({ showRedirectModal }: Props) {
 
   const poolsIncentivizedLabel = useMemo(() => {
     if (arbitrumIncentiveState?.lp?.isActive && avalancheIncentiveState?.lp?.isActive) {
-      return t`Arbitrum and Avalanche GM Pools are incentivized.`;
+      return (
+        <Trans>
+          <ExternalLink href={ARBITRUM_INCENTIVES_V2_URL}>Arbitrum</ExternalLink> and{" "}
```

Comment:
let's replace all such places with `getIncentivesV2Url(ARBITRUM)` and same for avalanche

Reply:
ok done